app/controllers/FeedbackFormsController.scala

```diff
@@ -109,14 +111,20 @@ class FeedbackFormsController @Inject()(val messagesApi: MessagesApi,
     }
   }
 
-  def delete(id: String): Action[AnyContent] = AdminAction.async { implicit request =>
-    feedbackRepository
-      .delete(id)
-      .map(_.fold {
-        Logger.error(s"Could not delete feddback form for id, $id")
-        Redirect(routes.FeedbackFormsController.manageFeedbackForm(1)).flashing("message" -> "Something went wrong!")
-      }(_ =>
-        Redirect(routes.FeedbackFormsController.manageFeedbackForm(1)).flashing("message" -> "Feedback form deleted successfully")))
+  def getFeedbackFormPreview: Action[JsValue] = AdminAction.async(parse.json) { implicit request =>
```

Comment:
Get feedback form id as a part of URI as added in a comment in conf/routes file.